tests/configs/example-config-custom-input-multi.yaml

```diff
@@ -0,0 +1,35 @@
+timeseries:
+  - name: custom-input-example4-multi
+    length: 5000 # must be equal to the length of the input timeseries
```

Comment:
Please allow this to be <= input timeseries length.

Please add an explicit check to the implementation and raise a `ValueError` if this condition is not met.